lib/faraday/response/raise_error.rb

```diff
@@ -38,7 +38,18 @@ def on_complete(env)
       end
 
       def response_values(env)
-        { status: env.status, headers: env.response_headers, body: env.body }
+        {
+          status: env.status,
+          headers: env.response_headers,
+          body: env.body,
+          request: {
```

Comment:
it's a bit weird to have "request" stuff in a method called "response_values" but at least they are namespaced, so maybe it's fine, let's see what the maintainers say

Reply:
Yeah, I agree, I think it would be cleaner if they were separated, but I didn't want to change too much of the code. As you say, it's probably up to the maintainers to decide what would be best, I wouldn't mind changing it.